src/SFA.DAS.Roatp.ProviderModeration.Web/Validators/ProviderDescriptionSubmitModelValidator.cs

```diff
@@ -9,11 +9,12 @@ public class ProviderDescriptionSubmitModelValidator : AbstractValidator<Provide
         public const string ProviderDescriptionHasInvalidCharacter = "Your answer must not include any special characters: @, #, $, ^, =, +, \\, /, <, >,";
         public const string ProviderDescriptionEmptyMessage = "Enter provider description";
         public const string ProviderDescriptionLengthErrorMessage = "Provider description must be 750 characters or less";
+        public const int ProviderDescriptionMaximumLength = 750;
         public ProviderDescriptionSubmitModelValidator()
         {
             RuleFor(x => x.ProviderDescription)
                 .NotEmpty().WithMessage(ProviderDescriptionEmptyMessage)
-                .MaximumLength(750).WithMessage(ProviderDescriptionLengthErrorMessage)
+                .Must(description => !string.IsNullOrEmpty(description) && description.Replace("\r","").Replace("\n","").Length<= ProviderDescriptionMaximumLength).WithMessage(ProviderDescriptionLengthErrorMessage)
```

Comment:
Mark, we already have an implementation in [apprentice web](https://github.com/SkillsFundingAgency/das-apprentice-aan-web/blob/3784a832ce98c94d66e9a06531f50190a1e64f3b/src/SFA.DAS.ApprenticeAan.Web/Validators/Onboarding/ReasonToJoinSubmitModelValidator.cs#L17) can you please see if this works? It surely looks prettier.

Reply:
That validator is for a word count, not a character count, so leaving this as is

Reply:
Your PR title says word count as well. So is this a character count?

Reply:
It was mis-named. I've changed the title to character